docs/assets/extra.css

```diff
@@ -1,3 +1,9 @@
 :root {
-    --md-primary-fg-color: #1c394a;
+    --md-primary-fg-color: #944038;
   }
+
+[data-md-color-scheme="slate"] {
+  --md-hue: 210;
+  /* for better contrast make the links bright */
+  --md-typeset-a-color: #E8A03E;
```

Comment:
It works very selectively I'd say. for now I'm [linking](https://github.com/squidfunk/mkdocs-material/blob/6b5035f5580f97532d664e3d1babf5f320e88ee9/src/assets/stylesheets/main/_colors.scss) their css colors template for possible future changes.

This issue https://github.com/squidfunk/mkdocs-material/issues/2987 might be helpful as well